spec/assimilate_spec.rb

```diff
@@ -77,7 +77,7 @@
     stubs.create_folder('athena', 'files', 'blueprints')
 
     # we try to assimilate the blueprints folder
-    @cmd.execute('athena', 'files', '/', 'spec/stubs/blueprints')
+    @cmd.execute('athena', 'files', '/', 'spec/data/stubs/blueprints')
```

Comment:
I noticed this path was incorrect; the stubs directory normally belongs under the data directory here, but some reason it was coded with this path.  The test passes with this change and fails without.